packages/api/test/__snapshots__/mutations.test.ts.snap

```diff
@@ -0,0 +1,91 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`\`validateCart\` mutation, returning full order 1`] = `
```

Comment:
Yep! Since in these tests we're working with data that might change in its source, snapshots should enable us to more easily validate when that happens and tests start failing. We're performing real queries and fetching data from the `storeframework`, so if something changes in a product, for example, the test might break. Snapshots are an easy way to catch these types of "errors" since we can just update the snapshot to account for the change in the data source.

The way this was done before, without snapshots, would require you to update a static file (`/mocks/*.ts`) with the expected query result. So you would need to run the local server, perform the query then copy and paste its result in a file. Snapshots just seem easier to maintain :)

Reply:
But when the product data change, the snapshot match will fail too... The problem remains

Reply:
Yes! But then you just need to run `yarn test -u` and the snapshot is updated, instead of having to start the local server, run the query, copy and paste the result into the mock file. It just makes it simpler